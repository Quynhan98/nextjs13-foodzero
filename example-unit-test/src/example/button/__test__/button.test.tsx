import * as renderer from 'react-test-renderer';
import { Button } from '..';

describe('Test Button component', () => {
  test('Component Button matches DOM Snapshot', () => {
    const component = renderer
      .create(
        <Button color={'btn-primary'} typeButton={'button'} size={'btn-small'}>
          Button
        </Button>
      )
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});
