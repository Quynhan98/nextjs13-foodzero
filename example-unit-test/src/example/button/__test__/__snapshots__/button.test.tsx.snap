// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Button component Component Button matches DOM Snapshot 1`] = `
<button
  className="button btn-primary btn-small"
  type="button"
>
  Button
</button>
`;
